app/src/ui/preferences/advanced.tsx

```diff
@@ -114,6 +127,37 @@ export class Advanced extends React.Component<
             onChange={this.onReportingOptOutChanged}
           />
         </div>
+        {(this.state.canUseWindowsSSH || enableExternalCredentialHelper()) && (
+          <h2>Network and credentials</h2>
+        )}
+        {this.renderSSHSettings()}
+        {enableExternalCredentialHelper() && (
+          <div className="advanced-section">
+            <Checkbox
+              label={'Use Git Credential Manager'}
+              value={
+                this.state.useExternalCredentialHelper
+                  ? CheckboxValue.On
+                  : CheckboxValue.Off
+              }
+              onChange={this.onUseExternalCredentialHelperChanged}
+              ariaDescribedBy="use-external-credential-helper-description"
+            />
+            <div
+              id="use-external-credential-helper-description"
+              className="git-settings-description"
+            >
+              <p>
+                Use{' '}
+                <LinkButton uri="https://github.com/git-ecosystem/git-credential-manager">
+                  Git Credential Manager{' '}
+                </LinkButton>{' '}
+                for private repositories outside of GitHub.com. This feature is
+                experimental and subject to change{' '}
```

Comment:
Nit: this can be removed, right?
```suggestion
                experimental and subject to change
```